static/src/js/components/CustomToggle/RadioSettingToggle.scss

```diff
@@ -15,10 +15,12 @@
     width: 0px;
     height: 0px;
     overflow: hidden;
-    font-size: 0.875rem;
+    font-size: 0.75rem;
+    font-weight: 700;
     color: utils.$color__carbon--80;
     margin-left: calc(bootstrap.$spacer / 4);
     text-indent: -10000px;
+    margin: 0;
```

Comment:
`margin-left: calc(bootstrap.$spacer / 4);` followed by `margin: 0;` seems wrong

Reply:
removed the margin-left